src/geometry/manifold/smooth_manifold_with_corners.lean

```diff
@@ -619,6 +619,50 @@ instance prod {𝕜 : Type*} [nondiscrete_normed_field 𝕜]
 
 end smooth_manifold_with_corners
 
+namespace topological_space.opens
+
+open topological_space
+
+variables  {𝕜 : Type*} [nondiscrete_normed_field 𝕜]
+  {E : Type*} [normed_group E] [normed_space 𝕜 E]
+  {H : Type*} [topological_space H] (I : model_with_corners 𝕜 E H)
+  {M : Type*} [topological_space M] [charted_space H M] [smooth_manifold_with_corners I M]
+  (s : opens M)
+
+instance : smooth_manifold_with_corners I s :=
+{ ..topological_space.opens.has_groupoid (times_cont_diff_groupoid ∞ I) s }
```

Comment:
```suggestion
s.has_groupoid (times_cont_diff_groupoid ∞ I)
```
does this work?

Reply:
Yes

Reply:
Can you please make the change then?  I mean specifically
```lean
s.has_groupoid (times_cont_diff_groupoid ∞ I)
```
without the braces

Reply:
No sorry, it does not work without the braces, I meant putting `s` at the start worked!